### PR TITLE
Add get block REST API

### PR DESCRIPTION
**Pull request by trevorbernard**
Fixes:
* Add Webserver (https://github.com/Granola-Team/mina-indexer/issues/375)
* add get block REST API (https://github.com/Granola-Team/mina-indexer/issues/368)

### CODE REVIEW

**Comment by trevorbernard** [comment]
I'm running into nix build woes. I can recreate on other environments.

File: src/web/rest/blocks.rs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    store::IndexerStore,
+};
+
+#[derive(Deserialize)]
+struct Params {
+    limit: Option<u32>,
+}
+
+fn get_limit(limit: Option<u32>) -> u32 {
+    limit.map(|value| value.min(10)).unwrap_or(1)
```
Comment (n1tranquilla) [inline src/web/rest/blocks.rs:22]: I know defaulting to 1 is at parity with current API, but should we default to something more reasonable?

**Review by n1tranquilla — APPROVED** [review]
LGTM

**Review by Isaac-DeFrain — APPROVED** [review]
This is passing CI and @n1tranquilla approves. I am optimistically merging

File: src/web/rest/blocks.rs (hunk 1 above)
Comment (Isaac-DeFrain) [reply src/web/rest/blocks.rs:22]: Agreed. This is a discussion that should happen. Resolving for now to merge

**Pull request merged**